src/main/java/athena/ui/AthenaUi.java

```diff
@@ -53,8 +53,8 @@ public void printTaskAdded(String taskName, String taskStartTime, String taskDur
         System.out.println("You've successfully added " + colorText.toBlue(taskName) + " to your list!\n"
                 + "It will start at " + colorText.toBlue(taskStartTime) + " and finish on "
                 + colorText.toBlue(taskDeadline) + ".\n"
-                + "You should spend a total of " + colorText.toBlue(taskDuration) + " on it.\n"
-                + "It is set to happen " + colorText.toBlue(taskRecurrence) + " and has an importance of "
+                + "You should spend a total of " + colorText.toBlue(taskDuration) + " hour(s) on it.\n"
+                + "It is set to happen every" + colorText.toBlue(taskRecurrence) + " and has an importance of "
```

Comment:
Add a check if the taskRecurrence is Today or a particular date, probably don't need "every today" to be printed out

Reply:
fixed!